### PR TITLE
Release 6.2.6

### PR DESCRIPTION
**Pull request by damacus**
Release 6.2.6

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          2 Errors
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please provide a summary of your Pull Request.</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please add labels to this Pull Request</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ This file is used to list changes made in each version of the haproxy cookbook.
 
 ## [unreleased]
 
+## [v6.2.6](2018-11-05)
+
+- Put use_backend rules before the http-request rules
```
Comment (JohnRoesler) [inline CHANGELOG.md]: It's putting the `http-request` rules before the `use_backend` so this statement should be flipped

Comment (damacus) [reply CHANGELOG.md]: Thanks for catching this. it was a bit late last night

**Comment by damacus** [comment]
Thanks @JohnRoesler this has now been released  to the supermarket

**Pull request merged**